tests/ci-commitmessage-submodules.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+BUMP_SUBMODULE_REGEXP="Update submodule: "
```

Comment:
This is a cool idea! Though we can make it even smarter/stronger 💪 , esp. to deal with multiple submodules (as we have here and in rpm-ostree). E.g.:

- Get the list of submodule subdirs present from `git submodule status` (ouhh actually there's an even nicer way: `git submodule foreach --quiet 'echo $path'`)
- Use e.g. `git diff --name-only` to check if a commit modifies a submodule path
- Grep for `^Update submodule: $submodule` in `git show -s --format=%B`